tests/bears/BearTest.py

```diff
@@ -518,6 +521,24 @@ def test_do_settings(self):
         with self.assertRaises(ValueError):
             Debugger(bear=None)
 
+    def test_is_debugged(self):
+        with self.assertRaises(ValueError):
+            _is_debugged(bear=None)
+
+        section = Section('name')
+        uut = Bear(section, self.queue)
+        self.assertEqual(_is_debugged(uut), False)
+        section.append(Setting('debug_bears', 'tRuE'))
+        self.assertEqual(_is_debugged(uut), True)
+        section.append(Setting('debug_bears', '0'))
+        self.assertEqual(_is_debugged(uut), False)
+        section.append(Setting('debug_bears', 'Bear, ABear'))
+        self.assertEqual(_is_debugged(uut), True)
+
+        section.append(Setting('debug_bears', 'abc, xyz'))
+        with self.assertRaises(ValueError):
```

Comment:
why is 'abc, xyz' a ValueError ?

that looks like a valid list of strings.

Reply:
Yeah it is valid list of string but it is not valid valid for `debug_bears` setting.

Reply:
If I understand correctly this is what Makman is saying ?https://github.com/coala/coala/pull/5677#discussion_r206652926

Reply:
Or I made a mistake IMO, this is what Makman wanted?
```python
    try:
        return bool(bear.section['debug_bears'])
    except ValueError:
        pass
    try:
        return (bear.name.lower() in (
                map(str.lower, bear.section['debug_bears']))
    except ValueError:
        raise ValueError('could not parse setting "debug_bears" into bool or '
                         'string-list')
```

Reply:
Yeah it will never through a `ValueError`

Reply:
Of course he doesnt want ValueError's thrown when there isnt a `ValueError`.
You merged the wrong two conditions, making the `bool()` one be caught and re-raised when it shouldnt have been re-raised.